app/src/main/java/com/example/countryinfo/CountryDetailsFragment.kt

```diff
@@ -77,7 +77,9 @@ class CountryDetailsFragment : Fragment(), CountryDetailsAdapter.ViewHolder.OnIt
     }
 
     private fun showDialog(section: DetailsSections, dialogMessage: List<String>) {
```

Comment:
[nit] We can refactor this as well and pass two strings for title and message.

Reply:
Done.